lib/generators/templates/trk_datatable_test.rb

```diff
@@ -0,0 +1,34 @@
+RSpec.describe <%= @trk_class_name %> do
```

Comment:
Instead of `RSpec.describe` in minitest, we can use
```
require "test_helper"

class <%= @trk_class_name %>Test < ActionDispatch::IntegrationTest
```

Reply:
Correct, I overlooked this when I used copy-paste....